### PR TITLE
min date condition fix

### PR DESCRIPTION
**Pull request by Aeonax**
🤦

### CODE REVIEW

File: src/DatePicker.js [hunk 1]
```diff
@@ -129,9 +129,9 @@ function Week({ days, minDate, onDateClick, selectedDate, isDateAvailable }: Wee
   return (
     <Grid type="xs">
       {days.map(day => {
-        const notWithinMinDate = !minDate || day.date < minDate;
+        const WithinMinDate = minDate && day.date < minDate;
```
Comment (fmnoise) [inline src/DatePicker.js:132]: `isWithinMinDate`

File: pages/index.js [hunk 2]
```diff
@@ -211,7 +212,7 @@ class DatePickerDemo extends React.Component {
 
   render() {
     const { selectedDate, showDatePicker } = this.state;
-    const today = (new Date).toISOString()
+    const today = dateformat(new Date, 'yyyy-mm-dd')
```
Comment (fmnoise) [inline pages/index.js:215]: is it really needed?

Comment (Aeonax) [reply pages/index.js:215]: yep, we send it to `DatePicker` in this format and it truly support only this format.

`(new Date).toISOString() === "2020-07-05T17:36.."` with it we will have `2020-07-05 < 2020-07-05T17:36...` and min date will be excluded too=(
I could create a new date from min date iso string then parse it to iso string of required format, but not for a fast fix... and it is so fat...

**Comment by fmnoise** [comment]
Looks good to me
@bskog what do you think?

**Pull request merged**